ruby/vampires.rb

```diff
@@ -28,12 +34,32 @@
 		puts "Please answer yes or no."
 	end
 
+
+puts "Please list your allergies and enter them one at a time. Type 'done' when you are finished."
+allergies = gets.chomp
+	if allergies == "sunshine"
+		sunshine_allergy = true	
+	end
+	loop do 
+		puts "Any other allergies?"
+		multiple_allergies = gets.chomp
+		if multiple_allergies == "done"
+			break
+		end
+	end		
+		
+
 if age == (current_year - birth_year) && (garlic_affinity || health_insurance_seeker)
```

Comment:
This structure does not test all of the conditions before it prints a result, for instance if everything else looks good and they would be labeled as 'probably not a vampire' but their name was 'Drake Cula', you would want to output "Definitely a Vampire!" 

Think about how you could break up this tree to have all of the clauses run and update a variable instead of directly printing to the console when a result is positive.